src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -358,10 +358,12 @@ public override ImmutableArray<CSharpAttributeData> GetAttributes()
 
         private TypeParameterBounds GetBounds(ConsList<TypeParameterSymbol> inProgress)
         {
-            Debug.Assert(!inProgress.ContainsReference(this));
-            Debug.Assert(!inProgress.Any() || ReferenceEquals(inProgress.Head.ContainingSymbol, this.ContainingSymbol));
+            // PROTOTYPE(NullableReferenceTypes): Re-enable asserts.
```

Comment:
>PROTOTYPE [](start = 15, length = 9)

Do you plan to re-enable in this PR?

Reply:
I was planning on addressing this when the PR was smaller. At this point, I'd prefer to handle this in a subsequent PR.

---
In reply to: [195546019](https://github.com/dotnet/roslyn/pull/27525#discussion_r195546019) [](ancestors = 195546019)